package.json

```diff
@@ -1265,6 +1263,7 @@
     "@octokit/types": "^5.1.0",
     "apollo-boost": "^0.1.22",
     "apollo-link-context": "^1.0.12",
+    "cross-fetch": "^3.0.5",
```

Comment:
@RMacfarlane FYI, since version 3.* of node-fetch remove browser support I had to add a depenency on cross-fetch. It's only used in the browser since I don't know if it will work exactly as expected everywhere.